litecommands-core/src/dev/rollczi/litecommands/argument/resolver/standard/InstantArgumentResolver.java

```diff
@@ -38,18 +33,30 @@ public InstantArgumentResolver(MessageRegistry<SENDER> messageRegistry) {
 
     @Override
     public ParseResult<Instant> parse(Invocation<SENDER> invocation, Argument<Instant> argument, RawInput rawInput) {
-        final String commandInput = String.join(RawCommand.COMMAND_SEPARATOR, rawInput.seeNext(2));
+        String commandInput = String.join(RawCommand.COMMAND_SEPARATOR, rawInput.seeNext(2));
 
         try {
-            final String dateInput = String.join(" ", rawInput.next(2));
-            return ParseResult.success(Instant.from(formatter.parse(dateInput)));
-        } catch (DateTimeException ignored) {
+            String rawInstant = String.join(RawCommand.COMMAND_SEPARATOR, rawInput.next(2));
+            Instant instant = Instant.from(formatter.parse(rawInstant));
+
+            return ParseResult.success(instant);
+        }
+        catch (DateTimeException ignored) {
             return ParseResult.failure(this.messageRegistry.getInvoked(LiteMessages.INSTANT_INVALID_FORMAT, invocation, commandInput));
         }
     }
 
+    @Override
+    public SuggestionResult suggest(Invocation<SENDER> invocation, Argument<Instant> argument, SuggestionContext context) {
+        return IntStream.range(0, 7)
```

Comment:
Should we cache that instead of creation every time?

Reply:
this is based on `Instant.now()`